pkgs/development/python-modules/torch/default.nix

```diff
@@ -150,6 +150,8 @@ in buildPythonPackage rec {
     # Propagate CUPTI to Kineto by overriding the search path with environment variables.
     # https://github.com/pytorch/pytorch/pull/108847
     ./pytorch-pr-108847.patch
+  ] ++ lib.optionals rocmSupport [
+    ./0000-rocm-suppress-spammy-warnings.patch
```

Comment:
The build system can be bypassed with

`NIX_CFLAGS_COMPILE`

ex: https://github.com/NixOS/nixpkgs/blob/bab997db72a94fd58c6d3864f71ad8a81544cc6a/pkgs/development/libraries/libdynd/default.nix#L18

There are a lot of warnings even without `rocmSupport` so it would be good to reduce those too.

Reply:
Thanks for pointing that out!